src/Api/Persistent.hs

```diff
@@ -241,11 +247,11 @@ findWildIdeasBySpace space = findAllIn dbIdeas (\idea -> idea ^. ideaSpace == sp
 
 -- | FIXME: anyone can login
 -- | FIXME: every login changes all other logins
-loginUser :: ST -> Persist ()
+loginUser :: UserLogin -> Persist ()
 loginUser login = modifyDb dbCurrentUser . const . fmap (view _Id) =<< findUserByLogin login
 
-logoutUser :: ST -> Persist ()
-logoutUser _login = modifyDb dbCurrentUser $ const Nothing
+logoutUser :: Persist ()
```

Comment:
I disagree here. The `currentUser` in the DB is a hack. We should not build our abstraction on it.